### DISCUSSION TITLE
CCLA Manager Program

### DISCUSSION THREAD

**Discussion by KelseyBlodgett** [rabbitmq/rabbitmq-server#2895] [Ideas]
Hello, 

I am a CLA manager for Amazon and the program you use to manage your CLA makes it extremely difficult for us to sign this. In order for this to be a valid signature for our company I need to send this to an authorized signatory with an approval of the document by multiple people. They then have to navigate through the signature process rather than just being sent an email with a link to sign.

Additionally, we would like to sign this by email domain, but some of our employees have addresses like amazon.fr etc and I assume we would need to have multiple CLAs for those entities. How does someone sign for an email domain that they do not have an email address in? We have one signatory for CCLAs and they only have an asmazon.com email address and thus would not be able to sign for someone in amazon.fr. 

Do you have any plans to make this easier for corporate entities in the future?

**Comment by michaelklishin** [comment]
Hi @KelseyBlodgett,

The CLA tool we use is broadly used in the open source world. Most of our community (who are individuals, not large corporations) find it drastically easier to use than sending scans of documents to an email address.
So I guess the ease of use of this tool is inversely proportional to the size of one's legal department.

Please contact `info@rabbitmq.com` and we will suggest what other options there are. There is a different process
that some corporations have gone though.

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(
